components/ShadowLayer/examples/ShadowCornerRadiusExample.m

```diff
@@ -104,6 +105,19 @@ - (void)viewDidLoad {
   [self.view addSubview:_shadowsView];
 }
 
+- (void)viewDidLayoutSubviews {
+  [super viewDidLayoutSubviews];
+
+  if (@available(iOS 11.0, *)) {
+    _shadowsView.frame = UIEdgeInsetsInsetRect(self.view.bounds, self.view.safeAreaInsets);
+  } else {
+    CGRect frame = self.view.bounds;
+    frame.origin.y += self.topLayoutGuide.length;
```

Comment:
Isn’t topLayoutGuide only in iOS 9?

Reply:
Introduced in iOS 7, deprecated in iOS 11.